src/test/java/com/incognia/api/IncogniaAPITest.java

```diff
@@ -127,6 +127,62 @@ void testRegisterSignup_whenDataIsValid() {
     assertThat(signupAssessment.getReasons()).containsExactly(expectedReason);
   }
 
+  @Test
+  @DisplayName("should return the expected signup response")
```

Comment:
nit: talvez ter um `displayName` nome que diferencie do teste da linha 68?

Reply:
sure, just forgot to change the name. fixing right now

Reply:
done